src/maskAnalysis/__tests__/getConvexHull.test.ts

```diff
@@ -1,5 +1,9 @@
+import { toBeDeepCloseTo, toMatchCloseTo } from 'jest-matcher-deep-close-to';
+
 import { getConvexHull } from '../getConvexHull';
 
+expect.extend({ toBeDeepCloseTo, toMatchCloseTo });
```

Comment:
This is should be done in `jest.setup.ts`.

Reply:
I have the impression that if I import in `jest.setup.ts` I don't need to do it at all in the test files, is this correct?

Reply:
yes